src/Sample/Program.cs

```diff
@@ -18,39 +19,32 @@ public static async Task Main(string[] args)
         await TranscribeFileUsingPlan(kernel);
     }
 
-    private static Kernel BuildKernel(IConfiguration config)
-    {
-        var kernel = Kernel.CreateBuilder()
-            .AddOpenAIChatCompletion(
-                "gpt-3.5-turbo",
-                config["OpenAI:ApiKey"] ?? throw new Exception("OpenAI:ApiKey configuration is required.")
-            )
-            .Build();
-
-        var apiKey = config["AssemblyAI:ApiKey"] ?? throw new Exception("AssemblyAI:ApiKey configuration is required.");
-
-        kernel.ImportPluginFromObject(
-            new TranscriptPlugin(apiKey: apiKey)
-            {
-                AllowFileSystemAccess = true
-            },
-            TranscriptPlugin.PluginName
-        );
-
-        kernel.ImportPluginFromType<FindFilePlugin>(FindFilePlugin.PluginName);
-        return kernel;
-    }
-
     private static IConfigurationRoot BuildConfig(string[] args)
     {
         var config = new ConfigurationBuilder()
+            .AddJsonFile("appsettings.json")
             .AddEnvironmentVariables()
             .AddUserSecrets<Program>()
             .AddCommandLine(args)
             .Build();
         return config;
     }
 
+    private static Kernel BuildKernel(IConfiguration config)
+    {
+        var kernelBuilder = Kernel.CreateBuilder();
+        kernelBuilder.Services.AddSingleton(config);
```

Comment:
I think Semantic Kernel should integrate the .NET configuration system more out of the box so users don't have to build their own configuration unless they want to, and users don't have to add the configuration to the service collection themselves.